consensus/istanbul/core/handler.go

```diff
@@ -193,7 +199,26 @@ func (c *core) handleCheckedMsg(msg *message, src istanbul.Validator) error {
 	return errInvalidMessage
 }
 
-func (c *core) handleTimeoutMsg() {
+func (c *core) handleTimeoutMsg(view *istanbul.View) {
```

Comment:
`nextView` or `targetView`?
```suggestion
func (c *core) handleTimeoutMsg(nextView *istanbul.View) {
```

Reply:
Applied, thanks.